app/web_framework/cluster_base_handler.py

```diff
@@ -38,8 +38,10 @@ def _handle_request_exception(self, ex):
         :param ex: The exception that was caught
         :type ex: Exception
         """
-        # _handle_request_exception() is called in the exception handler, so we can still use logger.exception.
-        self._logger.exception('Exception occurred during request to {}.', self.request.uri)
+        # Requests to access an item that isn't ready shouldn't log anything
+        if type(ex) is not ItemNotReadyError:
```

Comment:
While you're here, do you think there are any other exception types we should remove from logging? 404s seem like they'd be another good one to omit. Thoughts?